modules/ocf_desktop/manifests/init.pp

```diff
@@ -22,4 +22,27 @@
   include ocf_desktop::udev
   include ocf_desktop::wireshark
   include ocf_desktop::xsession
+
+  # Allow HTTP and HTTPS
+  include ocf::firewall::allow_http
+
+  # Allow Steam login and Steam content
+  ocf::firewall::firewall46 {
+    '101 allow steam (tcp)':
+      opts => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => ['27015-27030', 27036, 27037],
```

Comment:
Nitpick: I would opt for consistence - if one of these has to be a string, then make these all strings. Same with below.